### PR TITLE
Delete spinner

### PR DESCRIPTION
**Pull request by BrannanC**
# Description

Adds a spinner when deleting a credential, cleans up card and uses dynamic date

## Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: client/src/components/Dashboard/Card/CredCard.jsx [hunk 1]
```diff
@@ -6,13 +6,22 @@ import CredCardSchoolName from './CredCardSchoolName';
...
 import CredCardDeleteBtn from './CredCardDeleteBtn';
-import { Button, Layer } from 'grommet';
+import { Layer } from 'grommet';
 import emblem from '../../../images/certEmblem.png';
 
 const CredCard = ({ cred }) => {
   const [show, setShow] = useState();
-  const { credName, criteria, ownerName, id, credHash } = cred;
-  console.log(credHash);
+  const {
```
Comment (nwthomas) [inline client/src/components/Dashboard/Card/CredCard.jsx]: Nice use of destructuring! ❄️

File: client/src/components/Dashboard/Card/CredCard.jsx [hunk 2]
```diff
@@ -6,13 +6,22 @@ import CredCardSchoolName from './CredCardSchoolName';
...
+    credName,
+    criteria,
+    ownerName,
+    id,
+    credHash,
+    issuedOn,
+    description,
+    imageUrl
+  } = cred;
+  console.log(cred);
```
Comment (meganjeffcoat) [inline client/src/components/Dashboard/Card/CredCard.jsx]: ✨

File: client/src/components/Dashboard/Card/CredCard.jsx [hunk 3]
```diff
@@ -24,22 +33,22 @@ const CredCard = ({ cred }) => {
...
               <h3>Issued by: [School of the Sequoias]</h3>
-              <h2>{cred.ownerName}</h2>
+              <h2>{ownerName}</h2>
             </section>
           </CertificateArea>
         </Layer>
       )}
       <CredCardSchoolName credName={credName} criteria={criteria} />
-      <CredCardDateIssued />
+      <CredCardDateIssued date={issuedOn} />
```
Comment (nwthomas) [inline client/src/components/Dashboard/Card/CredCard.jsx]: Thanks for fixing this! ⛄️

Comment (meganjeffcoat) [inline client/src/components/Dashboard/Card/CredCard.jsx]: 👍🏻

File: client/src/components/Dashboard/Card/CredCardDateIssued.jsx [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const CredCardDateIssued = () => {
+const CredCardDateIssued = ({ date }) => {
   return (
     <DateIssuedContainer>
-      <p>2/2/2018</p>
+      <p>{date}</p>
```
Comment (nwthomas) [inline client/src/components/Dashboard/Card/CredCardDateIssued.jsx]: Dynamic date = great fix 🌨

File: client/src/components/Dashboard/Card/CredCardDeleteBtn.jsx [hunk 5]
```diff
@@ -54,9 +59,15 @@ const CredCardDeleteBtn = ({ credId, credHash }) => {
...
       <CredCardDelBtnContainer>
         <CredCardDeleteButton
-          onClick={() => setHasActiveConfirmationDialog(true)} // This state value setting will cause the layer to appear
+          onClick={
+            isDeleting ? null : () => setHasActiveConfirmationDialog(true)
+          } // This state value setting will cause the layer to appear
         >
-          <TrashButton />
+          {isDeleting ? (
+            <Loader type="Oval" color="#d8d8d8" height="30" width="30" />
```
Comment (nwthomas) [inline client/src/components/Dashboard/Card/CredCardDeleteBtn.jsx]: _Excellent_ refactor. Very nice touch. Thanks for doing this. ☃️

File: client/src/components/Dashboard/Card/CredCardDateIssued.jsx (hunk 4 above)
Comment (meganjeffcoat) [inline client/src/components/Dashboard/Card/CredCardDateIssued.jsx]: 🎉

**Pull request merged**